CODEOWNERS

```
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @sameo @c3d @dcmiddle @magowan @bpradipt @jodh-intel @Jakob-Naucke
```

How about changing this to the [doc team](https://github.com/orgs/confidential-containers/teams/documentation-maintainers/members) as that avoids hard-coding users and also makes it easier to manage as we don't need to modify the in-tree doc:

```suggestion
*.md       @documentation-maintainers
```

---

Thanks, I changed to the doc team, syntax recommended here -> https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners
suggested to use the format @org/team

I have left the file pattern as * simply to ensure LICENSE and CODEOWNERS files gets included or indeed any other file formats which would get added. Open to discussion if you have other thoughts?